code/modules/mining/equipment/regenerative_core.dm

```diff
@@ -42,7 +42,8 @@
 		go_inert()
 
 /obj/item/organ/regenerative_core/proc/preserved(implanted = 0)
-	inert = FALSE
+	if(inert)
+		return
```

Comment:
Might wanna give some feedback to the player about the stabilizer being consumed without any effect or just not allow the stabilizer to be used on inert cores now that I think of it